src/models/analytics-query.ts

```diff
@@ -41,6 +52,7 @@ const schema = Joi.object().keys({
     }).required(),
   )).required(),
   filter: conditionSchema,
+  limiter: limiterSchema,
```

Comment:
While I went through https://github.com/withthegrid/platform/pull/936 I noticed that the wording is different from what we discussed earlier: `limiter` instead of `limitBy`. Why do you think the former is more fitting? It sounds more active than the other properties.

Reply:
it is consistent with the filter which is not called filterBy

Reply:
Fair enough, but wouldn't it then be `limit`?

Reply:
Is an option I considered. For me limiter would be an object that allows you to choose a limit and then limits for you. So to me it would be analogous to the machine that you use to configure the limit in your car. You can increase the limit set on the limiter, decrease the limit set on the limiter and the limiter will then make sure you will not drive faster than the limit configured. However if you strongly prefer limit or limitBy I'm happy to change it.

Reply:
For me limiter is the same kind of "machine" you describe. My only issue is that the other parameter names are not such machines.